src/views/GeneralConfig.vue

```diff
@@ -109,6 +109,75 @@
 						</openwb-base-alert>
 					</div>
 				</div>
+				<div v-if="!installAssistantActive">
+					<openwb-base-button-group-input
+						title="HTTP-API"
+						:buttons="[
+							{
+								buttonValue: false,
+								text: 'Aus',
+								class: 'btn-outline-danger',
+							},
+							{
+								buttonValue: true,
+								text: 'An',
+								class: 'btn-outline-success',
+							},
+						]"
+						:model-value="
+							$store.state.mqtt['openWB/general/http_api']
+						"
+						@update:model-value="
+							updateState('openWB/general/http_api', $event)
+						"
+					>
+						<template #help>
+							Mit der HTTP-API kann man den Wert eines MQTT-Topics
+							per HTTP GET oder POST abfragen. Ein Zugriff ist nur
+							lesend möglich, nicht schreibend.<br />
+							GET-Request:
+							<a
+								:href="`http://${getIpAddress()}:8080/?topic=openWB/system/time`"
+								target="_blank"
+								rel="noopener noreferrer"
+							>
+								http://{{
+									getIpAddress()
+								}}:8080/?topic=openWB/system/time </a
+							><br />
+							Verschlüsselter GET-Request:
+							<a
+								:href="`https://${getIpAddress()}:8443/?topic=openWB/system/time`"
+								target="_blank"
+								rel="noopener noreferrer"
+							>
+								https://{{
+									getIpAddress()
+								}}:8443/?topic=openWB/system/time
+							</a>
+							<br />
+							POST-Request mit curl:
+							<openwb-base-copy-to-clipboard
+								class="text-info"
+								tooltip="Topic kopieren"
+								>curl -X POST --data "topic=openWB/system/time"
+								http://{{
+									getIpAddress()
+								}}:8080/</openwb-base-copy-to-clipboard
+							>
+							Verschlüsselter POST-Request über curl mit privatem
+							SSL-Zertifikat:
+							<openwb-base-copy-to-clipboard
+								class="text-info"
+								tooltip="Topic kopieren"
+								>curl -k -X POST --data
+								"topic=openWB/system/time" https://{{
+									getIpAddress()
+								}}:8080/</openwb-base-copy-to-clipboard
+							><br />
```

Comment:
Ursache ist in https://github.com/openWB/openwb-ui-settings/pull/421 beschrieben. Begonnene Korrektur in https://github.com/openWB/openwb-ui-settings/pull/422.

Mögliche Formatierung, ohne auf den anderen PR warten zu müssen:
```suggestion
							<p>
								Mit der HTTP-API kann man den Wert eines
								MQTT-Topics per HTTPs (Port 8443) oder optional
								auch HTTP (Port 8090) abfragen. Ein Zugriff ist
								nur lesend möglich, nicht schreibend. Es wird
								empfohlen, nur HTTPs zu aktivieren.
							</p>
							<p>
								Beispiel: Abfrage des aktuellen
								Systemzeitstempels
							</p>
							<ul>
								<li>
									Verschlüsselter GET-Request:
									<a
										:href="`https://${getIpAddress()}:8443/?topic=openWB/system/time`"
										target="_blank"
										rel="noopener noreferrer"
									>
										https://{{
											getIpAddress()
										}}:8443/?topic=openWB/system/time
									</a>
								</li>
								<li>
									GET-Request (falls unverschlüsseltes HTTP
									aktiviert wurde):
									<a
										:href="`http://${getIpAddress()}:8080/?topic=openWB/system/time`"
										target="_blank"
										rel="noopener noreferrer"
									>
										http://{{
											getIpAddress()
										}}:8080/?topic=openWB/system/time
									</a>
								</li>
							</ul>
							<p>
								Die zurückgegebenen Daten sind im JSON-Format.
								Der Inhalt ist nach folgendem Schema aufgebaut:
							</p>
							<pre>
{
	"topic": "openWB/system/time",
	"value": 1721287000.646975,
	"status": "success"
}
							</pre>
							<p>
								Im Fehlerfall wird der Status auf "error"
								gesetzt und eine Fehlermeldung zurückgegeben.
							</p>
							<pre>
{
	"topic": "openWB/unbekanntes/topic",
	"value": null,
	"status": "error",
	"error": "Topic not found"
}
							</pre>
```
Die Beschreibung zu POST habe ich bewusst entfernt.